core/src/test/java/org/apache/iceberg/metrics/TestScanMetricsResultParser.java

```diff
@@ -61,15 +64,39 @@ public void missingFields() {
                 "{\"total-planning-duration\":{\"count\":3,\"time-unit\":\"hours\",\"total-duration\":10}}"))
         .isEqualTo(
             new ScanMetricsResult(
-                totalPlanningDuration, null, null, null, null, null, null, null, null));
+                totalPlanningDuration,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
```

Comment:
I completely agree with your observation, since it just adds visual noise. In fact https://github.com/apache/iceberg/pull/5780 will provide exactly that functionality out of the box. I'm planning to get https://github.com/apache/iceberg/pull/5780 first in and then I'll be rebasing this PR.

Reply:
should be fixed now